Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -129,23 +180,39 @@ extension BRBViewController: BRBConnectionDelegate {
         }
     }
     
-    func loginFailed(with error: String) {
-        loginViewController.errorDescription = error
-        loginViewController.setShowErrorMessage(true, animated: true)
-        loginViewController.isLoading = false
+    func failedToGetAccount(with error: String) {
+        isLoading = false
+        if loggedIn {
+            showErrorAlert(error: error)
+        } else {
+            loginViewController.errorDescription = error
+            loginViewController.setShowErrorMessage(true, animated: true)
+            loginViewController.isLoading = false
+        }
     }
     
 }
 
 extension BRBViewController: AboutTableViewControllerDelegate {
-    
+
     func aboutTableViewControllerDidLogoutUser(_ stvc: AboutTableViewController) {
         setState(.login)
-        
-        connectionHandler = BRBConnectionHandler()
-        connectionHandler.delegate = self
-        
+        UserDefaults.standard.set(nil, forKey: "BRBAccount")
```

Comment:
Clearing of cache should be handled by `BRBAccountManager` b/c how the information is cached should be invisible to the rest of the application